akka-http2-support/src/main/scala/akka/http/scaladsl/model2/FrameEvent.scala

```diff
@@ -0,0 +1,3 @@
+package akka.http.scaladsl.model2
+
+trait FrameEvent
```

Comment:
Do we want this to be public API right now? I think we can keep this in the `impl` package private for now.